5calls/app/src/main/java/org/a5calls/android/a5calls/view/OutcomeView.java

```diff
@@ -0,0 +1,105 @@
+package org.a5calls.android.a5calls.view;
+
+import android.content.Context;
+import android.content.res.Configuration;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.AppCompatButton;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.LinearLayout;
+
+import org.a5calls.android.a5calls.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class OutcomeView extends LinearLayout {
```

Comment:
Why do so much layout work in Java instead of XML? I'd much prefer XML layouts, I find them easier to read and edit than Java (plus Android Studio has a good preview feature).

Reply:
Using a RecyclerView now. See:

https://github.com/5calls/android/pull/89/files#diff-4b1b18290a53a1a020c32ca84f6efc7dR1
https://github.com/5calls/android/pull/89/files?diff=split#diff-619663bd5226fe79ebf27b7e14359d94R197
https://github.com/5calls/android/pull/89/files?diff=split#diff-6d9976551a58cdfcf44f3b6a3c33c12aR1